lint.go

```diff
@@ -1159,11 +1159,17 @@ func (f *file) lintErrors() {
 	}
 }
 
-func lintCapAndPunct(s string) (isCap, isPunct bool) {
+func lintCapAndPunct(s string) (isCap, isPunct, isNewline bool) {
 	first, firstN := utf8.DecodeRuneInString(s)
 	last, _ := utf8.DecodeLastRuneInString(s)
 	isPunct = last == '.' || last == ':' || last == '!'
+	isNewline = last == '\n'
 	isCap = unicode.IsUpper(first)
+	if isNewline && strings.Index(s, "\n") != len(s)-1 {
+		// Don't flag strings that contain more than one newline. They
```

Comment:
Hey Dominik, thanks for pulling at that data you mentioned. I'm inclined to think that the exception should not exist.

Could you upload a file somewhere (GitHub gist?) that contains all of the offending lines so I can take a look?